### ISSUE TITLE
[Suggestion] Implementation with Remnant.wiki

### ISSUE THREAD

**Issue by ben132** [Razzmatazzz/RemnantSaveGuardian#233]
This idea is to have links and details from the Remnant.wiki site.  Maybe even the Remnant2Toolkit

**Comment by crackedmind** [comment]
It's already here. Mouse right click in Missing Items or Possible Items Columns, also works on Missing items tab
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/1381503/b3a725c3-5ddf-4e9c-88bc-c8e95053e70d)
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/1381503/c0187da1-9581-4c76-bee0-cbdccebcf15c)

**Comment by TheNasbit** [comment]
> It's already here. Mouse right click on Missing Items or Possible Items Columns, also works on Missing items tab [...]

That is not what they meant - there is a new "official" wiki __NOT from fextralife__, that's work in progress (also with support from gunfire games)
Wiki used in past/current: [Fextralife](https://remnant2.wiki.fextralife.com/)
New Wiki: [remnant.wiki](https://remnant.wiki/)

I changed from fextralife to remnant.wiki in my PR, with an open TODO to add an option, so everyone can choose which one they prefer.

**Comment by AndrewSav** [comment]
IMO fextralife is more complete and the navigation is way better. What are advantages of the new one?

> also with support from gunfire games

How do we know this?

**Comment by TheNasbit** [comment]
> IMO fextralife is more complete and the navigation is way better. What are advantages of the new one?

Yes, fextralife is more complete for now - but that's mainly because the "new wiki" is ... new (just a few months). But the community is working hard to get it on the same level - or better, without plain copying fextralife.

Advantages? I guess it comes down to whatever people prefer to use. IMO it's good to give the choice to the user, so everyone can use what they prefer to use.

That said: I added the new wiki as an option in the settings.

**Comment by AlexijSeptimus** [comment]
> IMO fextralife is more complete and the navigation is way better. What are advantages of the new one?
> 
> > also with support from gunfire games
> 
> How do we know this?

Wiki mods have NDAs with Gunfire and Tragic (one of the prime devs) is supporting it.
